protected/humhub/tests/codeception/_support/HumHubHelperTrait.php

```diff
@@ -0,0 +1,666 @@
+<?php
+
+/*
+ * @link      https://www.humhub.org/
+ * @copyright Copyright (c) 2023 HumHub GmbH & Co. KG
+ * @license   https://www.humhub.com/licences
+ */
+
+namespace tests\codeception\_support;
+
+use Codeception\Exception\ModuleException;
+use Codeception\Module;
+use Codeception\Module\Yii2;
+use humhub\components\behaviors\PolymorphicRelation;
+use humhub\helpers\DbHelper;
+use humhub\models\UrlOembed;
+use humhub\modules\activity\models\Activity;
+use humhub\modules\content\widgets\richtext\converter\RichTextToHtmlConverter;
+use humhub\modules\content\widgets\richtext\converter\RichTextToMarkdownConverter;
+use humhub\modules\content\widgets\richtext\converter\RichTextToPlainTextConverter;
+use humhub\modules\content\widgets\richtext\converter\RichTextToShortTextConverter;
+use humhub\modules\notification\models\Notification;
+use PHPUnit\Framework\Constraint\LogicalNot;
+use PHPUnit\Framework\Exception;
+use TypeError;
+use Yii;
+use yii\base\ErrorException;
+use yii\base\InvalidConfigException;
+use yii\db\ActiveRecord;
+use yii\db\ExpressionInterface;
+use yii\helpers\FileHelper;
+use yii\log\Dispatcher;
+
+/**
+ * Humhub Test Helper Functions
+ *
+ * @since 1.15
+ */
+trait HumHubHelperTrait
```

Comment:
Why use a trait here?

Reply:
Because the classes 
- `\tests\codeception\_support\HumHubHelper` and
- `\tests\codeception\_support\HumHubDbTestCase`
do share already a common set of functions but do not derive from the same parent classes.

Particularly, when it comes to cleaning the cache, it is useful to have only one place where things need to be amended. That's why the trait has been introduced in 
- #6516

Also, it is good to have all the assert* functions available in both cases. So that's why I have now added/moved all those functions into the trait. This reduces the codebase to maintain.